### ISSUE TITLE
suggestions

### ISSUE THREAD

**Issue by itcarroll** [itcarroll/RE-NOBM-PCC#4]
## while preparing for ECSF at GSFC

- [ ] more, more relevant (i.e. Rrs), and more independent training data
  - [ ] finish pipeline to get monthly Rrs
  - [ ] get daily rrs with Cecile's help
- [ ] multiple outputs for presence vs abundance, weighting abundance by presence
- [ ] check for quality of data with OC4 algorithm on total chlorophyll
- [ ] pca on inputs (as an experiment), and maybe outputs too?

## mtg w/ ML division head at HRT

- [ ] tweaks: test sigmoid activation, magic learning rate (3e-4)
- [ ] consider binning and classification
- [ ] check for any association in data with 1 vs 2 nearest neighbors
- [ ] convolution was a bad idea, come up with another "experiment" for a paper (possibly multiple outputs)

##
